devops/files/decoupledREADME.md

```diff
@@ -0,0 +1,9 @@
+# Composer-enabled Decoupled WordPress template
```

Comment:
Is there a standard where `decoupledREADME.md` is the established filename? I feel like this could/should be something else (e.g. `decoupled.md`, `decoupled-readme.md`, etc or even `decoupled-README.md`). The weird camelCASE is bothering me. 😅

Reply:
I pulled this from our implementation in [pantheon-systems/drupal-composer-managed](https://github.com/pantheon-systems/drupal-composer-managed/pull/3/commits/f66dcb9c88a93ec8139ea5e1b72d45a5141d8c59), but don't have an issue with changing the name.

Reply:
I think `decoupled-README.md` makes more sense and we could update it over there as well for consistency.